docs/resources/index.md

```diff
@@ -1,25 +1,83 @@
+::::{grid}
+:gutter: 1
 
-We host a series of 8 to 10 weekly planning sessions on zoom during the two months prior to a hackweek. These sessions include:
+:::{grid-item-card} [Tutorial Lead](#tutorial-lead)
+[![GH](https://img.shields.io/static/v1?label=eScience&message=Tutorial-Lead&color=b7a57a)](#tutorial-lead)
+:::
 
-* a kickoff meeting to build team connections and to get clear on the purpose of the hackweek
-* scheduling and logistics planning 
-* tutorial ideation and design 
-* website creation and publishing of Jupyter Notebooks
-* designing structures to support project team work
+:::{grid-item-card} [Project Coordinator](#project-coordinator)
+[![GH](https://img.shields.io/static/v1?label=eScience&message=Project-Coordinator&color=85754d)](#project-coordinator)
+:::
 
-## Professional Development Opportunities
+:::{grid-item-card} [Helper](#helper)
+[![GH](https://img.shields.io/static/v1?label=eScience&message=Helper&color=4b2e83)](#helper)
+:::
 
-We recognize that many people will be new to tutorial design and guiding project work. Therefore we offer several opportunities for professional development for hackweek organizing teams. These include training sessions in:
+::::
+
+## Tutorial Lead Meeting Schedule
+
+Tutorial leads and developers will gather two months prior to a hackweek to create content and prepare for teaching. Here is the weekly schedule:
+
+```{Admonition} Week 1 / 90 min: Kick-off Meeting (everyone)
+Create tutorial schedule outline, form tutorial teams, and get to know each other.
+```
+
+```{Admonition} Week 2: Video Training (individual)
+:class: tip
+**Tutorial Development (20 min)**: Best practices, pedagogy, learning outcomes, and scoping a tutorial.
+
+**Technical Infrastructure (20 min)**: Overview of Jupyter Notebooks and pushing content to our Jupyter Book.
+```
+
+```{Admonition} Week 2 (60 min): Tutorial Outlining Meeting (tutorial teams)
+Meet with tutorial team to scope and outline tutorial content.
+```
+
+```{Admonition} Week 3 (60 min): Tutorial Report-Out Meeting (everyone)
+Each team shares scope and outline and together we identify overlaps, ways to integrate content and decide on tutorial sequencing.
+```
+
+```{Admonition} Week 3: Open Office Hours (individual)
+:class: tip
+Get help with technical infrastructure and practice doing a pull request to the Jupyterbook.
+```
+
+```{Admonition} Week 4: Video Training (individual)
+:class: tip
+**Tutorial Development (20 min)**: Learn ways to engage with participants and incorporate peer-learning into your lessons.
+```
+
+```{Admonition} Week 4 (60 min): Tutorial Development (tutorial teams)
+Discuss places/ways to engage participants and incorporate interactive components.
+```
+
+```{Admonition} Week 5: Open Office Hours (individual)
+:class: tip
+Tutorial development help.
+```
+
+```{Admonition} Week 6 and 7 (60 min): Tutorial Feedback (everyone)
+Practice tutorial delivery and get constructive feedback from others on the team.
```

Comment:
```suggestion
Practice tutorial delivery and get constructive feedback from others on the organizing team.
```

Trying to make sure our use of "team" is clear whether we mean an individual tutorial team, the whole tutorial and education team, or the whole organizing team.